File: runtime_permission/src/main/java/net/crystalapps/permission/runtime/library/callbacks/PermissionSettingCallback.java
package net.crystalapps.permission.runtime.library.callbacks;

import android.support.annotation.NonNull;

import net.crystalapps.permission.runtime.library.models.Perm;

/**
 * Created by Syed Owais Ali on 10/13/2018.
 */
public interface PermissionSettingCallback<T> {
    void onGranted(@NonNull Perm<T> perm);
    void onDenied(@NonNull Perm<T> perm);
}

File: mint-runtime-permission/src/main/java/net/crystalapps/permission/runtime/library/callbacks/PermissionSettingCallback.kt
package net.crystalapps.permission.runtime.library.callbacks

import net.crystalapps.permission.runtime.library.models.Perm

/**
 * Created by Syed Owais Ali on 10/13/2018.
 */
interface PermissionSettingCallback<T> {
    fun onGranted(perm: Perm<T>)
    fun onDenied(perm: Perm<T>)
}
